src/pages/shop/Shop.js

```diff
@@ -0,0 +1,242 @@
+                    <img src="" alt=""></img>
+                    <img src="" id="firstClone" alt=""></img>
+                </div>
+            </div>
+
+            
+                {/* signature collection section*/}
+                <div className="signature-container">
+                    <div className="signature-first-container">
+                        <div className="signature-header-container">
+                            <header className="signature-header">
+                                <p>Signature Collection</p>
+                            </header>
+                            <div className="signature-description">
+                                <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit "</p>
+                            </div>
+                        </div>
+                        <button className="learnMore-button">Learn More</button>
+                    </div>
+                    <div className="signature-second-container">
+                        <div className="sun">
+                            <img className="Sun-image" src={} alt="sun" />
+                            <img className="SunSquare-image" src={} alt="sun square" />
+                        </div>
+                        <div className="moon">
+                            <img className="MoonCresent-image" src={} alt="moon cresent" />
+                            <img className="MoonCircle-image" src={} alt="moon circle" />
+                            <img className="MoonSquare-image" src={} alt="moon square" />
+                        </div>
+                    </div>
+                
+                </div>
+
+                <div className="products-container">
+                    {/* set section*/}
+                    <div className="set-container">
+                        <img className="set-image" src={} alt="set image" />
+                        <div className="set-second-half">
+                            <div className="set-header-container">
+                                <header className="set-header">
+                                    <p>Day & Night Cream Set</p>
+                                </header>
+                                <div className="set-price-container">
+                                    <p>"$55"</p>
+                                </div>
+                            </div>
+                            <div className="set-description">
+                                <div className="set-description-text">
+                                    <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."</p>
+                                </div>
+                                <div className="set-reviews-container">
+                                    <div className="set-stars-container">
+                                        <img className="star1-image" src={} alt="star 1" />
```

Comment:
making things like the stars container a reusable component could come in handy later down the road